src/components/InLineNotification.tsx

```diff
@@ -52,17 +55,21 @@ export function InLineNotification({
         {label}
       </Text>
     )
-  const Icon = severityIcons[severity]
+  const Icon = variantIcons[variant]
 
   return (
     <View
-      style={[styles.container, { backgroundColor: severityColor.secondary }, style]}
+      style={[styles.container, { backgroundColor: variantColor.secondary }, style]}
       testID={testID}
     >
       <View style={styles.row}>
-        <View style={styles.attentionIcon}>
-          <Icon color={severityColor.primary} size={20} />
-        </View>
+        {icon !== null && (
```

Comment:
nit: it seems like we're using `icon` to either provide a custom icon _or_ hide the default icon, to reduce surprises and implicit meanings in props, could we split this into 2 props? hideIcon and customIcon maybe?